src/libgv/GV_VecDir2_80016EF8.c

```diff
@@ -0,0 +1,11 @@
+#include <util/idaTypes.h>
+#include <sys/types.h>
+#include <libgte.h>
+
+extern ratan2_80094308(DWORD,DWORD);
+
+DWORD GV_VecDir2_80016EF8(SVECTOR* vec)
+{
+   DWORD ratan = ratan2_80094308(vec->vx,vec->vz);
+   return ratan & 0xfff;
```

Comment:
Can you try the return type as short? I think these masks are implicitly put by the compiler for an int -> short cast

Reply:
It's not that (0xfff instead of 0xffff), what do?

Reply:
ignore me and carry on :(